lib/cli.rb

```diff
@@ -544,6 +544,8 @@ def self.parse_scopes(scope_string)
       desc: "Show diffs of configuration files changes."
     c.switch "html", required: false, negatable: false,
       desc: "Open system description in HTML format in your web browser."
+    c.switch "verbose", required: false, negatable: false,
+      desc: "Show the list of filters that is applied before showing the description."
```

Comment:
I would rephrase it to get rid of the singular plural confusion and to keep the same tense as in all other descriptions.
`Show the filters that were applied ...`